CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # Change Log
 
+## [0.17.0](https://github.com/confluentinc/ksql/releases/tag/v0.17.0-ksqldb) (2021-03-31)
+
+### Features
+
+* add functions as UDF input types ([#6955](https://github.com/confluentinc/ksql/pull/6955)) ([1b39ab5](https://github.com/confluentinc/ksql/commit/1b39ab56360612d385554d61c137814655dbf4b9))
+* add lambda syntax to grammar ([#6868](https://github.com/confluentinc/ksql/pull/6868)) ([dd3f365](https://github.com/confluentinc/ksql/commit/dd3f365eba725c77cca63f9196dac174394ecb58))
+* adding TypeContext for Lambdas ([#6989](https://github.com/confluentinc/ksql/pull/6989)) ([1538b3a](https://github.com/confluentinc/ksql/commit/1538b3ad7975adbc1ad0a452a7538ad7f3f748ba))
+* Adds ability to bypass cache for pull queries ([#6891](https://github.com/confluentinc/ksql/pull/6891)) ([4b3bc96](https://github.com/confluentinc/ksql/commit/4b3bc96830fbea8051cfaf4e9d27c4399032813f))
+* Adds table scans to pull queries ([#6726](https://github.com/confluentinc/ksql/pull/6726)) ([ae14fca](https://github.com/confluentinc/ksql/commit/ae14fca76089b85653de15003aead0dae4d2a5de))

```

Comment:
nit: this duplicates the below - I recommend keeping just the "generic WHERE clauses" one